assets/js/components/GoogleChart.js

```diff
@@ -88,7 +89,7 @@ export default function GoogleChart( props ) {
 
 			setChart( googleChart );
 		}
-	}, [ loading, !! chartRef.current, visualizationLoaded, !! chart ] );
+	}, [ loading, visualizationLoaded, chart, chartID, chartType, onReady ] );
```

Comment:
There are some dependency arrays in the code which have values that don't look like React best practices, and eslint gives all sorts of warnings about these now (not having complex values in the dependencies array, not having unused values), so I'm following eslint's recommendations